versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/tutorials/deploying-the-iperf-application.mdx

```diff
@@ -220,98 +226,99 @@ To verify our iPerf installation, first switch contexts to the cluster with the
 Perform these steps:
 
 1. Switch the context to the cluster where you installed the Iperf Sleep.
-```
-kubectx <cluster name>
-```
+   ```
+   kubectx <cluster name>
+   ```
 
 2. To validate the iperf-sleep pods belonging to the namespace iperf, run the following command:
-```
-$ kubectl get pods -n iperf
-```
-Example Output
-```
-NAME                           READY   STATUS    RESTARTS   AGE
-iperf-sleep-5477bf94cb-vmmtd   4/4     Running   0          10s
-```
+   ```
+   kubectl get pods -n iperf
+   ```
+   Example Output
+   ```
+   NAME                           READY   STATUS    RESTARTS   AGE
+   iperf-sleep-5477bf94cb-vmmtd   4/4     Running   0          10s
+   ```
 
 3. Validate the ServiceImport using the following command:
-```
-$ kubectl get serviceimport -n iperf
-```
-Example Output
-```
-NAME           SLICE   PORT(S)   ENDPOINTS   STATUS
-iperf-server   lion              1           READY
-```
+   ```
+   kubectl get serviceimport -n iperf
+   ```
+   Example Output
+   ```
+   NAME           SLICE   PORT(S)   ENDPOINTS   STATUS
+   iperf-server   lion              1           READY
+   ```
 
 ## Validating the iPerf Server Installation
 Perform these steps:
 
 1. Switch the context to the cluster where you have installed the Iperf Server.
```

Comment:
can we change `Iperf` to `iPerf` if that's not the name of the server?

Reply:
updated. Also updated the sequence of the steps.